pkg/cmd/grafana-server/server.go

```diff
@@ -193,7 +204,9 @@ func (g *GrafanaServerImpl) Shutdown(reason string) {
 	g.shutdownFn()
 
 	// wait for child routines
-	g.childRoutines.Wait()
+	if err := g.childRoutines.Wait(); err != nil {
+		g.log.Error("Failed waiting for services to shutdown", "err", err)
```

Comment:
Think we only want to care if there's error which is not context cancelled, see https://golang.org/pkg/context/#pkg-variables

Currently now when exiting Grafana I get the following error logs which doesn't add that much of important information:
```
INFO[10-10|10:53:46] Stopped RenderingService                 logger=server reason="context canceled"
INFO[10-10|10:53:46] Stopped HTTPServer                       logger=server reason=nil
EROR[10-10|10:53:46] Failed waiting for services to shutdown  logger=server err="context canceled"
EROR[10-10|10:53:46] A service failed                         logger=server err="context canceled"
```